llama_stack/apis/memory_banks/memory_banks.py

```diff
@@ -144,3 +144,15 @@ async def register_memory_bank(
         provider_id: Optional[str] = None,
         provider_memory_bank_id: Optional[str] = None,
     ) -> MemoryBank: ...
+
+    @webmethod(route="/memory_banks/update", method="POST")
+    async def update_memory_bank(
+        self,
+        memory_bank_id: str,
+        params: BankParams,
+        provider_id: Optional[str] = None,
+        provider_memory_bank_id: Optional[str] = None,
```

Comment:
we should probably not have this right now

Reply:
we should just not have an update API because we cannot support complex "re-ingestion" semantics.